03 - Showcases & Templates/Dashboards/🗂️ Dashboards.md

```diff
@@ -9,6 +9,8 @@ tags:
 
 #placeholder/description
 
+[@Rainbell's Obsidian Homepage](https://github.com/Rainbell129/Obsidian-Homepage)
```

Comment:
Couple of things.

Maybe start a bullet list?

Also, I tried looking at that site and was not able to understand most of it... Might it be worth adding a little bit of description about what the Homepage provides, or why people should look at it, or how they can use it?

This would allow people to know in advance what they are looking at...

Reply:
I dunno, a bullet point for just one item feels a bit wrong, but I don't mind, I'll add it.

Also, I do not use that dashboard myself, so I wouldn't feel confident enough to describe it's (rather extensive looking) feature set. However, I do think that the description on their GitHub repo appear to be enough to get what it's doing?

Reply:
OK thanks, I don't feel strongly about the comments...

I'll go ahead and approve and merge.